### PR TITLE
Add 1.15.0 release notes and announcement

### PR DESCRIPTION
**Pull request by Monkeyanator**
Adds 1.15.0 release notes and announcement. Seems that no upgrade notes were added for 1.15.

### CODE REVIEW

File: content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+- **Added** `--log_output_level` and `--log_as_json` to the `istio-init` container (as they are in `istio-proxy`).
+
+- **Updated** the default value of the feature flag `ENABLE_LEGACY_FSGROUP_INJECTION` to false.
+This may cause issues with sidecars when installing on Helm on Kubernetes versions prior to 1.19.
+
+- **Updated** the Kiali addon to the latest version (v1.55).
+
+- **Added** support for watching local secret resource updates for external istiod.
+  ([Issue #31946](https://github.com/istio/istio/issues/31946))
```
Comment (frankbu) [inline content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md:111]:
```suggestion
- **Added** support for watching local secret resource updates for external istiod.
  ([Issue #31946](https://github.com/istio/istio/issues/31946))

- **Improved** [external control plane setup instructions](/docs/setup/install/external-controlplane/), including tips for simpler control plane ingress setup, making it easier to experiment with the external control plane deployment model in a test environment.
```

**Comment by ericvn** [comment]
We are still missing the page based on the Google Doc.

File: content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  ([Issue #37415](https://github.com/istio/istio/issues/37415))
+
+- **Added** `istioctl analyze` beta API version support for file resources.
+
+- **Added** pod name and cluster name to bookinfo's reviews, where the cluster name is determined by the `CLUSTER_NAME` environment variable on the reviews deployments.
+
+- **Added** support for parsing list type of files in `istioctl analyze`.
+  ([Issue #39982](https://github.com/istio/istio/issues/39982))
+
+- **Added** description to `istioctl admin log`
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md:135]: Add a period to the send of the sentence.

File: content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+- **Improved** the number of pushes to gateway proxies by not pushing when services are not visible from the gateway.
+  ([Issue #39110](https://github.com/istio/istio/issues/39110))
+
+- **Improved** compatibility with minimal host operating systems without `nsenter` binary (like Talos OS). The `cni.conf` flag `HostNSEnterExec` reverts to old behavior with use of nsenter.
+  ([Issue #38794](https://github.com/istio/istio/issues/38794))
+
+- **Updated** istiod to allow unknown flags for backward-compatibility. If an unknown flag is passed, no warning or error will be logged.
+
+- **Added** add a validation warning when protocol is unset and address is also unset.
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md:23]: ~~add~~ don't need double add.

Comment (Monkeyanator) [reply content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md]: Done.

File: content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md (hunk 2 above)
Comment (Monkeyanator) [reply content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md]: Done.

File: content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md (hunk 1 above)
Comment (Monkeyanator) [reply content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md:111]: Done.

**Comment by ericvn** [comment]
One thing I noticed is the items aren’t sorted by Action under each topic. Normally its the Added, then Deprecated continuing alphabetically up to Upgraded. See https://istio.io/latest/news/releases/1.14.x/announcing-1.14/change-notes/ for an example. You can certainly order in other ways, like what's maybe a higher priority, and that may have been done already.

**Comment by ericvn** [comment]
> Seems that no upgrade notes were added for 1.15.

I also looked and didn't see any upgradeNotes

**Comment by Monkeyanator** [comment]
> I also looked and didn't see any upgradeNotes. This may mean we only have something like:

Added blank upgrade notes section with that text.

>  You can certainly order in other ways, like what's maybe a higher priority, and that may have been done already.

Yeah I'd shuffled a bit based on importance. Reshuffled so that the verbs are together.

> We are still missing the page based on the Google Doc.

Added

File: content/en/docs/releases/supported-releases/index.md [hunk 4]
```diff
@@ -51,18 +51,19 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
...
-| 1.8             | No                   | November 10, 2020 | May 12, 2021             | 1.16, 1.17, 1.18, 1.19         | 1.15                         |
-| 1.7             | No                   | August 21, 2020   | Feb 25, 2021             | 1.16, 1.17, 1.18               | 1.15                         |
-| 1.6 and earlier | No                   |                   |                          |                                |                              |
+| Version         | Currently Supported  | Release Date      | End of Life              | Supported Kubernetes Versions | Tested, but not supported    |
+|-----------------|----------------------|-------------------|--------------------------|-------------------------------|------------------------------|
+| master          | No, development only |                   |                          |                               |                              |
+| 1.15            | Yes                  | August 31, 2022   | ~March 2023 (Expected)   | 1.22, 1.23, 1.24, 1.25        | 1.16, 1.17, 1.18, 1.19, 1.20 |
```
Comment (ericvn) [inline content/en/docs/releases/supported-releases/index.md:57]: Missing 1.21 in last column .

File: content/en/news/releases/1.15.x/_index.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+layout: release-grid
+decoration: dot
+---
+
+We are pleased to announce the release of Istio 1.15!
+
+{{< relnote >}}
+
+This is the third Istio release of 2022. We would like to thank the entire Istio community
+for helping to get Istio 1.15.0 published. Special thanks are due to the release managers Sam Naser and Aryan Gupta from Google, Ziyang Xiao from Intel and Daniel Hawton from solo.io. As always, our gratitude goes to Test & Release WG lead Eric Van Norman (IBM) for his help and guidance.
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/_index.md:15]: I expect solo.io here should be Solo.io or just Solo. I'm not sure how they differentiate whether.io should be added. I do see on their website, `At Solo.io, we have helped thousands of organizations` which means that they typically capitalize it unless it's the URL...

Comment (ericvn) [reply content/en/news/releases/1.15.x/_index.md:15]: @dhawton Thoughts on Solo vs Solo.io?

File: content/en/news/releases/1.15.x/_index.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+---
+title: 1.15.x Releases
+description: Announcements for the 1.15 release and its associated patch releases.
+weight: 15
+list_by_publishdate: true
+layout: release-grid
+decoration: dot
+---
+
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/_index.md:9]: The first 8 lines need to stay here, but the rest of the doc goes into content/en/news/releases/1.15.x/announcing-1.15/_index.md (so one level deeper in the tree).

You can copy the first few lines from 1.14 for example. I don't think you need aliases at this point.

File: content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
+---
+title: Istio 1.15.0 Change Notes
+linktitle: 1.15.0
+subtitle: Minor Release
+description: Istio 1.15.0 change notes.
+publishdate: 2022-08-31
+release: 1.15.0
+weight: 10
+aliases:
+    - /news/announcing-1.15.0
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md:10]: You don't need the alias.

Comment (Monkeyanator) [reply content/en/news/releases/1.15.x/announcing-1.15/change-notes/index.md]: Removed.

File: content/en/news/releases/1.15.x/_index.md (hunk 6 above)
Comment (Monkeyanator) [reply content/en/news/releases/1.15.x/_index.md]: Fixed file structure and moved one level deeper.

File: content/en/docs/releases/supported-releases/index.md (hunk 4 above)
Comment (Monkeyanator) [reply content/en/docs/releases/supported-releases/index.md]: Added.

File: .spelling [hunk 8]
```diff
@@ -855,6 +860,7 @@ Snell-Feikema
 SNI
 SolarWinds
 Solo.io
+solo.io.
```
[the remark this answers is not in the document]
Comment (Monkeyanator) [reply .spelling]: Removed line (swapped for `Solo.io` on the page)

**Review by ericvn — APPROVED** [review]
Merge this in preparation for the branch cut.

File: content/en/news/releases/1.15.x/_index.md (hunk 5 above)
Comment (dhawton) [reply content/en/news/releases/1.15.x/_index.md]: Solo.io is preferred.

**Pull request merged**